data/human/transport missions.txt

```diff
@@ -3209,3 +3209,76 @@ mission "Earth Retirement"
 			label reject
 			`	Charles looks surprised for a moment, and then, for the first time you've seen, he smiles. "Thank you, Captain <last>. I wish there were more people like you in the Paradise Worlds."`
 			`	You wish the couple the best of luck on Earth, and they slowly make their way to the train station. Presumably, their new apartment is far from here.`
+
+
+
+mission "Spacediving professionals"
+	name "Spacediving at <destination>"
+	description "Drop skydivers from the edge of space. They are carrying cooled spacesuits, durable parachutes and oxygen bottles to ensure the survival of their party. Payment is <payment>."
+	passengers 5 10
+	repeat
+	job
+	source
+		government "Republic" "Free Worlds" "Syndicate"
+	destination
+		distance 7 9
+		government "Republic" "Free Worlds" "Syndicate"
```

Comment:
These missions probably need to check whether their destination has enough atmosphere to parachute on, or if their destination is a planet at all, and not a station.
```suggestion
		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
```

Reply:
Agreed, need to add checks if the destination is suitable for parachuting.

Reply:
I added two checks for this.